### PR TITLE
Added support for cross compiling using MinGW on a Linux host.

### PR DESCRIPTION
**Pull request by mitza-oci**

### CODE REVIEW

File: ACE/ace/config-win32-mingw64.h [hunk 1]
```diff
@@ -127,6 +127,8 @@
 #define ACE_HAS_NONCONST_WCSDUP
 #define ACE_ISCTYPE_EQUIVALENT ::_isctype
 
+#define ACE_HAS_PTHREAD_SIGMASK_PROTOTYPE
```
Comment (jwillemsen) [inline ACE/ace/config-win32-mingw64.h]: Does this work with MinGW-64 on native windows?

Comment (mitza-oci) [reply ACE/ace/config-win32-mingw64.h]: I thought the header "/usr/share/mingw-w64/include/pthread_signal.h" would be the same as in the MinGW install on windows but I haven't checked yet.

Comment (mitza-oci) [reply ACE/ace/config-win32-mingw64.h]: I looked at the headers, I don't see any reason why it wouldn't work.

Comment (jwillemsen) [reply ACE/ace/config-win32-mingw64.h]: There is a build on the scoreboard, that will validate this

**Pull request merged**